### PR TITLE
BASE: Button Add Syncfusion

### PR DESCRIPTION
**Pull request by stasnowak**
Change the Button Base to be from Syncfusion library

#15

### CODE REVIEW

File: Taarafo.Portal.Web/Views/Bases/ButtonBase.razor [hunk 1]
```diff
@@ -1,3 +1,7 @@
-﻿<button @onclick="@Click" disabled="@IsDisabled">
-    @Label
-</button> 
\ No newline at end of file
+﻿@using Syncfusion.Blazor.Buttons
+
+<SfButton OnClick=@Click
+		  Disabled=@IsDisabled
+		  IsPrimary=@true>
```
Comment (hassanhabib) [inline Taarafo.Portal.Web/Views/Bases/ButtonBase.razor:5]: Fix the indentation here please

**Comment by hassanhabib** [comment]
@stasnowak you've got some conflicts here as well.

**Comment by cjdutoit** [comment]
@hassanhabib are we OK to use Syncfusion?  I see on nuget "This is a commercial product and requires a paid license for possession or use."

**Comment by hassanhabib** [comment]
@cjdutoit yeah there's a community license that we can use for this project. and it's for free for open source non-profile projects like OtripleS - so we're good. You can also generate your own key so you don't see this warning for free just make a Syncfusion account and you'll be just fine.

**Pull request merged**